src/config.cpp

```diff
@@ -672,6 +696,27 @@ CParseResult CConfig::parseLine(std::string line, bool dynamic) {
         if (ISVARIABLE)
             return parseVariable(LHS, RHS, dynamic);
 
+        // Removing escape chars. -- in the future, maybe map all the chars that can be escaped.
+        // Right now only expression parsing has escapeable chars
+        const char ESCAPE_CHAR = '\\';
+        const char ESCAPE_SET[]{'{', '}'};
+        for (long i = 0; i < (long)RHS.length() - (long)1; i++) {
+            if (RHS.at(i) != ESCAPE_CHAR)
+                continue;
+            //if escaping an escape, remove and skip the next char
+            if (RHS.at(i + 1) == ESCAPE_CHAR) {
+                RHS.erase(i, 1);
+                continue;
+            }
+            //checks if any of the chars were escapable.
+            for (unsigned int j = 0; j < sizeof(ESCAPE_SET) / sizeof(char); j++) {
```

Comment:
this is very C. Instead, make escape_set a `std::array` and use a foreach loop `for (const auto& c : escape_set)`